Author: Bernhard M. Wiedemann <bwiedemann suse.de>
Date: 2017-10-24

have fixed build dates
to make packages like rhino build more reproducibly
that use their build.xml to insert build dates into output files

Index: apache-ant-1.9.9/src/script/ant
===================================================================
--- apache-ant-1.9.9.orig/src/script/ant
+++ apache-ant-1.9.9/src/script/ant
@@ -290,6 +290,9 @@ fi
 if $usejikes; then
   ANT_OPTS="$ANT_OPTS -Dbuild.compiler=jikes"
 fi
+if test -n "$SOURCE_DATE_EPOCH" ; then
+  ANT_OPTS="$ANT_OPTS -Dant.tstamp.now=$SOURCE_DATE_EPOCH"
+fi
 
 # For Cygwin, switch paths to appropriate format before running java
 # For PATHs convert to unix format first, then to windows format to ensure
